### PR TITLE
Merge doc updates forward to Mitaka

### PR DESCRIPTION
**Pull request by jputrino**
@richbrowne @szakeri 

#### What issues does this address?
N/A

#### What's this change do?
Merges doc updates from liberty up to mitaka.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -158,9 +196,9 @@ See the License for the specific language governing permissions and
...
 
 Contributor License Agreement
-=============================
+-----------------------------
 
-Individuals or business entities who contribute to this project must have completed and submitted the `F5® Contributor License Agreement <http://f5-openstack-docs.readthedocs.org/en/latest/cla_landing.html#cla-landing>`_ to Openstack\_CLA@f5.com prior to their code submission being included in this project.
+Individuals or business entities who contribute to this project must complete and submit the `F5 Contributor License Agreement <http://f5-openstack-docs.readthedocs.org/en/latest/cla_landing.html#cla-landing>`_ to Openstack\_CLA@f5.com **before** their code submission can be added to this project.
 
 
 .. |Build Status| image:: https://travis-ci.org/F5Networks/f5-openstack-agent.svg?branch=liberty
```
Comment (szakeri) [inline README.rst]: I think branch should be mitaka

Comment (szakeri) [inline README.rst]: Also here

Comment (szakeri) [inline README.rst]: Also line 205, 211, 212

File: docs/README.rst [hunk 2]
```diff
@@ -0,0 +1,734 @@
...
+   .. code-block:: bash
+
+      $ cp ~/f5-upgrade-temp/f5-openstack-agent.ini /etc/neutron/services/f5/f5-openstack-agent.ini
+
+
+.. rubric:: Footnotes
+.. [#neutroncli] See the `Neutron LBaaS documentation <https://docs.openstack.org/mitaka/networking-guide/config-lbaas.html>`_
+.. [#agent] Similar to BIG-IP :term:`high availability`, but applies to the |agent-short| processes.
+
+.. |Build Status| image:: https://travis-ci.org/F5Networks/f5-openstack-agent.svg?branch=liberty
```
Comment (szakeri) [inline docs/README.rst]: branch name change to mitaka

File: docs/README.rst [hunk 3]
```diff
@@ -0,0 +1,734 @@
...
+.. _Neutron Modular Layer 2: https://wiki.openstack.org/wiki/Neutron/ML2
+.. _BIG-IP route domains: https://support.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/tmos-routing-administration-13-0-0/8.html
+.. _BIG-IP SNAT pools: https://support.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/tmos-routing-administration-13-0-0/7.html
+.. _OpenStack Barbican: https://wiki.openstack.org/wiki/Barbican
+.. _OpenStack Keystone: https://wiki.openstack.org/wiki/Keystone
+.. _Binds VLANs to BIG-IP interfaces: https://support.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/tmos-routing-administration-13-0-0/4.html
+.. _automap SNAT: https://support.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/tmos-routing-administration-13-0-0/7.html
+.. _configures the requested objects: /cloud/openstack/latest/lbaas/bigip-command-mapping.html
+.. _Distributed Virtual Router: https://specs.openstack.org/openstack/neutron-specs/specs/juno/neutron-ovs-dvr.html
+.. _Role Based Access Control: http://specs.openstack.org/openstack/neutron-specs/specs/liberty/rbac-networks.html
```
Comment (szakeri) [inline docs/README.rst]: branch name change to mitaka

File: README.rst [hunk 4]
```diff
@@ -1,7 +1,7 @@
 .. raw:: html
 
    <!--
-   Copyright 2015-2016 F5 Networks Inc.
+   Copyright 2015-2017 F5 Networks Inc.
```
Comment (szakeri) [inline README.rst]: Does this need to change to 2016-2017? I am not sure about how copyright works for F5

File: README.rst [hunk 5]
```diff
@@ -16,134 +16,172 @@
...
+Using the Built-in Debugger
+---------------------------
 
-If you find an issue, we would love to hear about it. Please open a new `issue <https://github.com/F5Networks/f5-openstack-agent/issues>`_ aissue for each bug you'd like to report or feature you'd like to request. Please be specific, and include as much information about your environment and the issue as possible.
+Use the built-in debugger -- ``debug_bundler.py`` -- to package information about your environment for debugging purposes.
 
-Contributing
-************
-See `Contributing <CONTRIBUTING.md>`_.
+When the you install ``f5-openstack-agent``, the ``debug_bundler.py`` script installs itself in ``/usr/bin/f5/``.
```
Comment (szakeri) [inline README.rst]: please remove "the" before "you install"

File: README.rst [hunk 6]
```diff
@@ -16,134 +16,172 @@
...
+
 
-If any issue is found with the debug_bundler script, please file an issue on GitHub.
 
 Copyright
-*********
+---------
 
-Copyright 2015-2016 F5 Networks Inc.
+Copyright 2015-2017 F5 Networks Inc.
```
Comment (szakeri) [inline README.rst]: also the copyright here

File: README.rst (hunk 4 above)
Comment (jputrino) [inline README.rst]: Yup - we need to be up-to-date with the current year.

File: docs/README.rst (hunk 3 above)
Comment (jputrino) [inline docs/README.rst]: For the RBAC link (`Role Based Access Control: http://specs.openstack.org/openstack/neutron-specs/specs/liberty/rbac-networks.html`) it needs to stay liberty -- this refers to the OpenStack version the feature was introduced in.

File: README.rst (hunk 6 above)
Comment (jputrino) [inline README.rst]: This needs to be here and is correct.

File: README.rst (hunk 5 above)
Comment (jputrino) [inline README.rst]: done

**Pull request closed without merging**